array.c

```diff
@@ -1765,17 +1911,49 @@ rb_ary_first(int argc, VALUE *argv, VALUE ary)
 
 /*
  *  call-seq:
- *     ary.last     ->  obj or nil
- *     ary.last(n)  ->  new_ary
+ *    array.last  -> object or nil
+ *    array.last(n) -> new_array
+ *
+ *  Returns elements from +self+; +self+ is not modified.
+ *
+ *  Argument +n+, if given, must be an
+ *  {Integer-convertible object}[doc/implicit_conversion_rdoc.html#label-Integer-Convertible+Objects].
+ *
+ *  ---
+ *
+ *  When no argument is given, returns the last element:
+ *    a = [:foo, 'bar', 2]
+ *    a.last # => 2
+ *    a # => [:foo, "bar", 2]
  *
- *  Returns the last element(s) of +self+. If the array is empty,
- *  the first form returns +nil+.
+ *  If +self+ is empty, returns +nil+:
+ *    [].last # => nil
  *
- *  See also Array#first for the opposite effect.
```

Comment:
Can we take this back? Same as the other references, as a non-native English speaker we may or may not what's the opposite of any given method.

Reply:
Done both for #first and #last.

Reply:
Thanks!